Editor/Scripts/CodeGenerators/RGActionCodeGenerator.cs

```diff
@@ -26,6 +29,13 @@ private static void GenerateRGScripts()
             WriteToJson("RGActions", actionJson);
             WriteToJson("RGStates", stateJson);
 
+        }
+
+        [MenuItem("Regression Games/Extract Data")]
+        private static void ExtractData()
+        {
```

Comment:
I agree with Zack re number of steps, let's just through this into generate scripts!

Reply:
Because objectType is set within the editor in the component? Or am I missing something....